cockroachcloud/serverless-faqs.md

```diff
@@ -79,11 +79,13 @@ Yes, your free cluster has been automatically migrated to {{ site.data.products.
 
 CockroachDB [automatically collects statistics](../{{site.current_cloud_version}}/cost-based-optimizer.html#table-statistics) in a background process when certain conditions are met (for example, when more than 20% of rows in a table are modified). The statistics are used by the cost-based optimizer to tune statements for higher performance.
 
-When automatic statistics collection starts your cluster may consume RUs above the 100 RUs per second baseline when your workload is otherwise consuming RUs below the baseline. You can [turn off automatic statistics collection](../{{site.current_cloud_version}}/cost-based-optimizer.html#enable-and-disable-automatic-statistics-collection-for-clusters) to avoid these RU bursts, but the cost-based optimizer may choose inefficient statement plans as it doesn't have access to the latest statistics.
+When automatic statistics collection starts you may see a spike in RU usage. You can [turn off automatic statistics collection](../{{site.current_cloud_version}}/cost-based-optimizer.html#enable-and-disable-automatic-statistics-collection-for-clusters) to avoid these RU bursts, but the cost-based optimizer may choose inefficient statement plans if it doesn't have access to the latest statistics.
+
+Refreshing the Cloud Console will also consume RUs.
```

Comment:
Since there could be a lot of clients consuming the Cloud Console, is there any way to help the customer predict how many RUs Cloud Console usage is likely to incur or to figure out if a single client or session is consuming more than expected?

Reply:
typically its the amount of RUs needed to list the sql users and databases in the cluster (since we populate that info in the connect modal for the cluster).

However, for the same point in the above comment i'd prefer not including numbers.

Reply:
Makes sense. It seems more that it is rendering Cloud Console pages than specifically refreshing Cloud Console.